iib/workers/tasks/build.py

```diff
@@ -162,28 +162,31 @@ def _finish_request_post_build(
     """
     conf = get_worker_config()
     if from_index and overwrite_from_index:
-        log.info(f'Ovewriting the index image {from_index} with {output_pull_spec}')
+        output_message = f'Ovewriting the index image {from_index} with {output_pull_spec}'
+        log.info(output_message)
         index_image = from_index
         exc_msg = f'Failed to overwrite the input from_index container image of {index_image}'
         args = [f'docker://{output_pull_spec}', f'docker://{index_image}']
         _skopeo_copy(*args, copy_all=True, dest_token=overwrite_from_index_token, exc_msg=exc_msg)
+        state_reason = output_message
     elif conf['iib_index_image_output_registry']:
         index_image = output_pull_spec.replace(
             conf['iib_registry'], conf['iib_index_image_output_registry'], 1
         )
-        log.info(
-            'Changed the index_image pull specification from %s to %s',
-            output_pull_spec,
-            index_image,
+        output_message = (
+            f'Changed the index_image pull specification from {output_pull_spec} to {index_image}'
         )
+        log.info(output_message)
+        state_reason = output_message
     else:
         index_image = output_pull_spec
+        state_reason = f'Updated the index image with {output_pull_spec}'
```

Comment:
This state reason is too verbose. I think it'd be nice to omit this from being a state reason update.